### PR TITLE
feat(494): add mc info in agenda

### LINKED ISSUE #494: Add MCs page
Something similar to the agenda? Is it possible to integrate to the agenda page?

### PR DESCRIPTION
**Pull request by TheJoin95**
Closes #494 

image

### CODE REVIEW

**Review by davideimola — DISMISSED** [review]

**Review by Wabri — CHANGES_REQUESTED** [review]
Leave this open until I get the last 2 confirmation!

**Comment by Wabri** [comment]
Would you mind to update the MCs with this?

Day 7 morning: Maciek Palmowski + Lorenzo Pieri
Day 7 afternoon: Irine Kokilashvili + Miki Lombardi
Day 8 morning: Noah Jelich + Miki Lombardi
Day 8 afternoon: Sabrina Mazzola + Lorenzo Pieri

File: pages/agenda.tsx [hunk 1]
```diff
@@ -57,13 +57,13 @@ export default function Agenda() {
           {trackSelected === 'Alpha1' && (
             <>
               <AgendaCard hour='8:30' isBreak={true} breakTitle='Open gate' icon='/icons/pencil.svg' position='right'/>
-              <AgendaCard hour='9:50' isBreak={true} breakTitle='Morning intro' icon='/icons/microphone.svg' position='left'/>
+              <AgendaCard hour='9:50' isBreak={true} breakTitle='Morning intro' subtitle='Presented by Lorenzo Pieri' icon='/icons/microphone.svg' position='left'/>
```
Comment (Wabri) [inline pages/agenda.tsx]:
```suggestion
              <AgendaCard hour='9:50' isBreak={true} breakTitle='Morning intro' subtitle='Presented by Lorenzo Pieri & Maciek Palmowski' icon='/icons/microphone.svg' position='left'/>
```

File: pages/agenda.tsx [hunk 2]
```diff
@@ -57,13 +57,13 @@ export default function Agenda() {
...
               <AgendaCard hour='10:00' isBreak={false} speakerName={s('pj.name')} talkTitle={s('pj.talk_title')} icon='/icons/microphone.svg' position='right' link={s('pj.id')}/>
               <AgendaCard hour='10:45' isBreak={false} speakerName={s('irine.name')} talkTitle={s('irine.talk_title')} icon='/icons/microphone.svg' position='left' link={s('irine.id')}/>
               {/* <AgendaCard hour='10:40' isBreak={true} breakTitle='Coffee break' icon='/icons/coffee.svg' position='left'/> */}
               <AgendaCard hour='11:30' isBreak={false} speakerName={s('iulia.name')} talkTitle={s('iulia.talk_title')} icon='/icons/microphone.svg' position='right' link={s('iulia.id')}/>
               <AgendaCard hour='12:15' isBreak={true} breakTitle='Lunch break' icon='/icons/fork-and-knife.svg' position='left'/>
-              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' icon='/icons/microphone.svg' position='right'/>
+              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' subtitle='Presented by Miki Lombardi & Maciek Palmowski' icon='/icons/microphone.svg' position='right'/>
```
Comment (Wabri) [inline pages/agenda.tsx]:
```suggestion
              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' subtitle='Presented by Miki Lombardi & Irine Kokilashvili' icon='/icons/microphone.svg' position='right'/>
```

File: pages/agenda.tsx [hunk 3]
```diff
@@ -99,13 +99,13 @@ export default function Agenda() {
...
               <AgendaCard hour='10:00' isBreak={false} speakerName={s('nathan.name')} talkTitle={s('nathan.talk_title')} icon='/icons/microphone.svg' position='right' link={s('nathan.id')}/>
               <AgendaCard hour='10:45' isBreak={false} speakerName={s('noam.name')} talkTitle={s('noam.talk_title')} icon='/icons/microphone.svg' position='left' link={s('noam.id')}/>
               {/* <AgendaCard hour='10:40' isBreak={true} breakTitle='Coffee break' icon='/icons/coffee.svg' position='left'/> */}
               <AgendaCard hour='11:30' isBreak={false} speakerName={s('christina.name')} talkTitle={s('christina.talk_title')} icon='/icons/microphone.svg' position='right' link={s('tbd.id')}/>
               <AgendaCard hour='12:15' isBreak={true} breakTitle='Lunch break' icon='/icons/fork-and-knife.svg' position='left'/>
-              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' icon='/icons/microphone.svg' position='right'/>
+              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' subtitle='Presented by Lorenzo Pieri' icon='/icons/microphone.svg' position='right'/>
```
Comment (Wabri) [inline pages/agenda.tsx]:
```suggestion
              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' subtitle='Presented by Lorenzo Pieri & Sabrina Mazzola' icon='/icons/microphone.svg' position='right'/>
```

**Review by Wabri — CHANGES_REQUESTED** [review]

**Review by Wabri — APPROVED** [review]

**Pull request merged**